### PR TITLE
Pluralize final summary report line

### PR DESCRIPTION
**Pull request by richardtemple**
This is my first real pull request, so please forgive me if I'm doing it wrong.  I just went to Ruby on Ales and was inspired by the talks (including zenspider's) to finally give contributing a go.  I wasn't sure where to put the utility method `#simple_pluralize`, so I left it under `#summary` for now.  I also see that there are some other areas where pluralization could be used, but I wanted to run this by seattlerb first before continuing incase there are good reasons to not pluralize reporting.  Also, any advice in helping me become a better contributor would be much appreciated.

### CODE REVIEW

File: lib/minitest.rb [hunk 1]
```diff
@@ -610,9 +610,26 @@ def summary # :nodoc:
...
+         simple_pluralize(errors, "error"),
+         simple_pluralize(skips, "skip"),
+         extra]
+    end
+
+    def simple_pluralize(n, singular, plural=nil)
+      if n == 1
+          "1 #{singular}"
+      elsif plural
+          "#{n} #{plural}"
```
Comment (zenspider) [inline lib/minitest.rb]: why is this section here if you don't use it?

**Comment by zenspider** [comment]
The main reason why I didn't do this already was just to maintain simplicity. I don't see anything overtly bad about this PR (tho I did add a q to the commit) but I don't know if I want to accept it for the reason mentioned.

**Comment by zenspider** [comment]
Where else do you think this should be applied?

File: lib/minitest.rb (hunk 1 above)
Comment (richardtemple) [inline lib/minitest.rb]: Do you mean the optional plural argument?  I saw it in another example and thought it made sense  if needed later.  I could take it out of course.

**Comment by richardtemple** [comment]
There was just one other place above in the `statistics` method.

**Comment by richardtemple** [comment]
Unfortunately, I'm not familiar with adding a q to a commit.

File: lib/minitest.rb (hunk 1 above)
Comment (zenspider) [inline lib/minitest.rb]: Yeah. The whole philosophy behind minitest is that "mini" part. It doesn't have any code it doesn't need. IOW: "If needed later" == "it isn't needed"

**Comment by zenspider** [comment]
You mean here: https://github.com/richardtemple/minitest/blob/master/lib/minitest.rb#L587

**Comment by richardtemple** [comment]
Cool github feature.  Yes, that was the method.  Thanks for being patient with my newbness.  (I've been using CVS and subversion doing Java and C# for a number of years.)

File: lib/minitest.rb (hunk 1 above)
Comment (richardtemple) [inline lib/minitest.rb]: Sounds extremely reasonable.  :)  I'll commit the change in my fork incase you're on the fence about merging it.

**Comment by zenspider** [comment]
To be fair... I'm still on the fence even with that cleanup.

**Comment by richardtemple** [comment]
Well, as much as I'd love to have my first accepted PR be for something as great as minitest, if this doesn't cut it, I've still learned more about working via github.  I really appreciate your candid comments.  Thank you!

**Comment by zenspider** [comment]
Yeah. Thank you very much for the contribution, but I'm going to decline.

**Comment by richardtemple** [comment]
Cool.  Thank you for considering.

**Pull request closed without merging**